src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/SVUtils.java

```diff
@@ -207,6 +88,11 @@ private void advance() {
         return Collectors.toCollection( () -> new ArrayList<>(size));
     }
 
+    // =================================================================================================================
+
+    //Workaround for seed 14695981039346656037 that doesn't fit in a signed long
+    private static final long FNV64_DEFAULT_SEED = new BigInteger("14695981039346656037").longValue();
```

Comment:
Curious what is the need to use a "problematic" seed number.

Reply:
not sure. That's a question for TedS.